.github/release-tool/changelog.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+type commit struct {
+	category string
+	message  string
+}
+
+func newCommit(line string) *commit {
+	var cat, msg string
+	lineSplit := strings.SplitN(line, ":", 2)
+	if len(lineSplit) > 1 {
+		cat = strings.TrimSpace(lineSplit[0])
+		msg = strings.TrimSpace(lineSplit[1])
+	} else {
+		cat = "other"
+		msg = strings.TrimSpace(lineSplit[0])
+	}
+	return &commit{category: cat, message: msg}
+}
+
+func (c *commit) String() string {
+	return c.message
+}
+
+type changelog struct {
```

Comment:
This could just be `type changelog map[string][]*commit`, which would simplify `newChangelog`

No biggie, though.

Reply:
I just copy-pasta'd this one, I'm going to leave as is if you don't mind :)